### ISSUE TITLE
[New Metric] Specificity

### ISSUE THREAD

**Issue by FlorianMF** [PyTorchLightning/metrics#106]
## 🚀 Feature
In addition to Precision and Recall it would be nice to have a Specificity metric.

For the implementation I think it would be enough to make a copy of Recall (class und function) and adapt numerator and denominator in _precision_compute.

### Alternatives
For binary classification Specificity is the same as Recall with 0 as true label. 
For multiclass classification this is not as easy as this though.

**Comment by arvindmuralie77** [comment]
@Borda , can I take up this issue?

**Comment by arvindmuralie77** [comment]
@Borda , for the test cases, since there is no specificity metric in sklearn, the randomized test cases cannot be used. Should I just write some test cases with fixed values?
